app/controllers/tournaments_controller.rb

```diff
@@ -49,10 +49,29 @@ def registration
     end
 
     if @tournament.nrdb_deck_registration?
-      begin
-        @decks = Nrdb::Connection.new(current_user).decks
-      rescue
-        redirect_to login_path(:return_to => request.path)
+      if @current_user_player.decks_locked?
+        corp_deck = @current_user_player.corp_deck
+        runner_deck = @current_user_player.runner_deck
+        if corp_deck.present?
+          @corp_deck = corp_deck.name
+          @corp_id_nrdb_code = corp_deck.identity_nrdb_printing_id
+        else
+          @corp_deck = 'No corp deck'
+          @corp_id_nrdb_code = '00005'
```

Comment:
add a comment after this saying what that card^Wprinting is for future readers?

Reply:
Good idea, thanks